src/main/kotlin/racing/service/CarService.kt

```diff
@@ -0,0 +1,22 @@
+package racing.service
+
+import racing.controller.dto.RoundResultView
+import racing.domain.Car
+import racing.domain.MovingRule
+
+class CarService(private val movingRule: MovingRule) {
+    fun moveCars(cars: List<Car>, round: Int): List<RoundResultView> {
+        val roundResultViews: MutableList<RoundResultView> = mutableListOf()
+        var racer: List<Car> = cars
+
+        for (i in 1..round) {
+            racer = movingRule.goForward(racer)
```

Comment:
racer라는 변수명만 보면 `List<Car>`가 아닌 Car라고 생각할 것 같아요.

```suggestion
        var racers: List<Car> = cars

        for (i in 1..round) {
            racers = movingRule.goForward(racer)
```